.ci/Jenkinsfile

```diff
@@ -39,7 +39,7 @@ pipeline {
     quietPeriod(10)
   }
   triggers {
-    issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?(?:benchmark\\W+)?tests(?:\\W+please)?.*')
+    issueCommentTrigger("(${obltGitHubComments()}|^run benchmark tests)")
```

Comment:
Is it a problem that the world `please` was removed from the end or will that still be accepted?

Reply:
It's accepted, see https://github.com/elastic/apm-agent-ruby/pull/1164#issuecomment-924832635 that triggered https://apm-ci.elastic.co/job/apm-agent-ruby/job/apm-agent-ruby-mbp/job/PR-1164/3/